jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -987,6 +987,35 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
 
         return parameters_cube
 
+    def get_model_parameters(self, model_label=None, x=None, y=None):
```

Comment:
Does this not need a change log?

Reply:
I don't think so (see explanation in description), but can add it if you think there should